### PR TITLE
Create homework_3

### PR DESCRIPTION
**Pull request by VladimirSkarb**

### CODE REVIEW

File: homeworks/vladimir.karbivnychiy_VladimirSkarb/homework_3/rec.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+const inputData = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+inputData.question('Введите минимальное значение: ', (min) => {
+    inputData.close();
+    let a = 1;
+    let b;
```
Comment (markoffden) [inline homeworks/vladimir.karbivnychiy_VladimirSkarb/homework_3/rec.js]: you do not need these variables, especially out your function scope

you just need to keep dividing by 2 until the number equals/not equals 1

make it simpler

File: homeworks/vladimir.karbivnychiy_VladimirSkarb/homework_3/number.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+function arraySq(array) {
+    return array.map(element => element * element);
+}
+console.log(arraySq(block(32, 4)));
+
+function sumArraySq(array) {
+    return array.reduce((sum, nextNum) => sum + nextNum);
+}
+console.log(sumArraySq(arraySq(block(32, 4))));
```
Comment (markoffden) [inline homeworks/vladimir.karbivnychiy_VladimirSkarb/homework_3/number.js]: here all is fine

File: homeworks/vladimir.karbivnychiy_VladimirSkarb/homework_3/rec.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        return rec(n);
+    }
+
+    for (let i = min; i <= 600; i++) {
+        b = rec(i);
+        if (b === true) {
+            console.log(i);
+        }
+    }
+});
```
Comment (markoffden) [inline homeworks/vladimir.karbivnychiy_VladimirSkarb/homework_3/rec.js]: again, avoid you functions to interact with data out their scope, unless it's necessary

for the second task, your script should consist of two units

1. recursive function taking one argument
2. loop utilizing this function

no extra variables outside, no side effects

**Comment by VladimirSkarb** [comment]
Доброе утро, я несколько дней назад писал в переписке, что уезжаю. Можно я по приезду переделаю и досдам? Спасибо.

**Comment by markoffden** [comment]
OK, hw3 is accepted, make sure you stay on track with the rest :)

**Pull request merged**